plenum/server/replica_validator_enums.py

```diff
@@ -1,9 +1,12 @@
 # ReplicaValidationResult
-PROCESS = 0
-DISCARD = 1
+from plenum.common import stashing_router
+
+PROCESS = stashing_router.PROCESS
+DISCARD = stashing_router.DISCARD
 STASH_VIEW = 2
 STASH_WATERMARKS = 3
 STASH_CATCH_UP = 4
+STASH_WAITING_NEW_VIEW = 5
```

Comment:
I think it's better to make base of stashing values more explicit, like:
```python
STASH_VIEW = stashing_router.STASH + 1
STASH_WATERMARKS = stashing_router.STASH + 2
...
```

Reply:
Agree, but please note that this stashing logic is WIP until Ordering Service integration is done (which has changes related to this as well).

Reply:
Ordering Service integration is done now ;)